test/framework/e2e.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 const (
-	defaultClusterConfigFile = "cluster.yaml"
-	defaultClusterName       = "eksa-test"
-	clusterNameVar           = "T_CLUSTER_NAME"
-	JobIdVar                 = "T_JOB_ID"
+	defaultClusterConfigFile         = "cluster.yaml"
+	defaultBundleReleaseManifestFile = "bin/local-bundle-release.yaml"
```

Comment:
A bit confused here.
Aren't we running the test from `bin`? Does the `--bundles-override` require an absolute path? 
But i do see that this worked on the presubmit.

Reply:
We're running the test from /home/e2e, and adding /home/e2e/bin to PATH